<launch>

    <arg name="dataset_path"    default="dataset.json" />
    <arg name="utterance_topic" default="utterance" />

	<node name="snips_nlu_ros" pkg="snips_nlu_ros" type="snips_nlu_node.py" output="screen">
        <param name='dataset' value="$(arg dataset_path)" type="string" />
        <remap from="utterance" to="$(arg utterance_topic)" />
    </node>
</launch>